modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/policy-navigator-details/history-info/index.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import PanelComponent from '../../../../../../common/components/panel';
+import {InfoPanelType} from '../types';
+
+type HistoryInfoType = {
+	arraySortedByDate: InfoPanelType[];
+	setShowPanel: (panel: boolean[]) => void;
+	showPanel: boolean[];
+};
+
+const HistoryInfo = ({
+	arraySortedByDate,
+	setShowPanel,
+	showPanel,
+}: HistoryInfoType) => {
+	const ContentDescription = ({description}: {description: string}) => (
+		<div className="d-flex justify-content-between">
+			<div>{description}</div>
+		</div>
+	);
```

Comment:
Move component to outside HistoryInfo.

Otherwise this Component will be re-created when the HistoryInfo re-renders